client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -135,17 +141,44 @@ class OrderFulfillment extends Component {
 		}
 	};
 
+	isAddressValidForLabels( address ) {
+		const { labelCountriesData } = this.props;
+		const countryData = labelCountriesData[ address.country ];
```

Comment:
Looks like `getCountriesData` could possibly return `null`, should we add a null check here?